src/trix/models/composition.coffee

```diff
@@ -478,6 +493,15 @@ class Trix.Composition extends Trix.BasicObject
   notifyDelegateOfInsertionAtRange: (range) ->
     @delegate?.compositionDidPerformInsertionAtRange?(range)
 
+  getPositionInBlock: ->
+    [startPosition, endPosition] = @getSelectedRange()
+    startLocation = @document.locationFromPosition(startPosition)
+    endLocation = @document.locationFromPosition(endPosition)
+    block = @document.getBlockAtIndex(endLocation.index)
+    return "end" if @getBlock()?.getConfig("breakOnReturn") and block.text.getStringAtRange([endLocation.offset, endLocation.offset + 1]) is "\n"
+    return "start" if @getBlock()?.getConfig("breakOnReturn") and block.text.getStringAtRange([endLocation.offset - 1, endLocation.offset]) is ""
```

Comment:
We’ve got calls like these all over the place:

```coffee
block.text.getStringAtRange([n, n + 1])
```

They’d be easier to read if we extracted a method on Text to get a single character at a given offset.